# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1543394855 -3600
#      Wed Nov 28 09:47:35 2018 +0100
# Node ID 64b972120ec37569e8a82992936c05129a8b8984
# Parent  494452db0477638177dc6805beac3651cd668355
doc: Mention example method "roots" in OOP example (bug #55120).

* doc/interpreter/oop.txi: Mention example method "roots" in OOP example.

diff --git a/doc/interpreter/oop.txi b/doc/interpreter/oop.txi
--- a/doc/interpreter/oop.txi
+++ b/doc/interpreter/oop.txi
@@ -145,17 +145,23 @@ function.
 
 @noindent
 To inquire whether a particular method exists for a user class, the
 @code{ismethod} function can be used.
 
 @DOCSTRING(ismethod)
 
 @noindent
-For example:
+For a polynomial class it makes sense to have a method to compute its roots.
+
+@example
+@EXAMPLEFILE(@polynomial/roots.m)
+@end example
+
+We can check for the existence of the @code{roots}-method by calling:
 
 @example
 @group
 p = polynomial ([1, 0, 1]);
 ismethod (p, "roots")
   @result{} 1
 @end group
 @end example
